index.html

```diff
@@ -32,134 +32,123 @@
-          <img src="images/Vector-VK.svg" alt="ВКонтакте" width="24" height="14">
-          <span class="visually-hidden">ВКонтакте.</span>
-        </a>
-      </li>
-      <li>
-        <a href="https://vk.com/htmlacademy">
-          <img src="images/Vector-facebook.svg" alt="Facebook" width="18" height="16">
-          <span class="visually-hidden">Фейсбук.</span>
-        </a>
-      </li>
-      <li>
+    <main class="main-page">
+      <section class="hero">
+        <img class="hero-logo" src="images/logo/Sedona.svg" width="458" height="352" alt="Добро пожаловать в Седону.">
+      </section>
+      <div class="footer-hero">
+        <h1>Седона — небольшой городок в Аризоне,<br>заслуживающий большего!</h1>
+        <p>Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Каньон!</p>
+      </div>
+      <section class="advantages">                      <!-- Преимущества -->
+        <h2 class="visually-hidden">Преимущества</h2>
+        <ul class="advantages-list">     <!-- 1строка -->
+          <li class="advantages-item blue">
+            <h3 class="advantages-title"> Настоящий <br> городок</h3>
+            <span class="advantages-number">-№1-</span>
+            <p class="advantages-text">Седона — не аттракцион для <br> туристов, там течёт своя <br> жизнь</p>
+          </li>
+          <li class="advantages-item">
+            <img src="images/advantages/photo-1.jpg" alt="Уютный дворик гостиницы." width="800" height="384">
+          </li>
+        </ul>
+        <ul class="advantages-list">     <!-- 2строка -->
+          <li class="advantages-item lightblue home">
+            <h3 class="advantages-title">Жильё</h3>
+            <p class="advantages-text">Рекомендуем пожить <br> в настоящем мотеле,<br>
+              всё как в кино!</p>
+          </li>
+          <li class="advantages-item food">
+            <h3 class="advantages-title">Еда</h3>
+            <p class="advantages-text">Всегда заказывайте<br>
+              топовый фирменный бургер, <br> вы не разочаруетесь!</p>
+          </li>
+          <li class="advantages-item lightblue present">
+            <h3 class="advantages-title">Сувениры</h3>
+            <p class="advantages-text">Не только китайского,<br>
+              но и настоящего местного <br> производства!</p>
+          </li>
+        </ul>
+        <ul class="advantages-list">      <!-- 3строка -->
+          <li class="advantages-item">
+            <img src="images/advantages/photo-2.jpg" alt="Пейзаж Седоны." width="800" height="384">
+          </li>
+          <li class="advantages-item blue">
+            <h3 class="advantages-title">Там есть <br> мост дьявола</h3>
+            <span class="advantages-number">-№2-</span>
+            <p class="advantages-text">Да, по нему можно пройти! <br> Если вы осмелитесь, <br> разумеется</p>
+          </li>
+        </ul>
+        <ul class="advantages-list">               <!-- 4строка -->
+          <li class="advantages-item lightblue">
```

Comment:
хорошее решение с классами для цвета квадратика
только обычно пишут к чему относится модификатор 
```suggestion
          <li class="advantages-item advantages-item-lightblue">
```